ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Subsystem50Parser.java

```diff
@@ -84,6 +87,10 @@ protected void readElement(final XMLExtendedStreamReader reader, final EJB3Subsy
                 parseEnableGracefulTxnShutdown(reader, ejb3SubsystemAddOperation);
                 break;
             }
+            case SERVER_INTERCEPTORS: {
+                parseServerInterceptors(reader, ejb3SubsystemAddOperation);
+                break;
+            }
             default: {
```

Comment:
So we are adding a new top-level element to ejb3 subsystem configuration.  How is the version and namespace evolved?  Do we need to bump it from 5.0 to 6.0, or to 5.1, or remain the same?  Since 5.0 is already in WF 16, I guess we cannot modify a released namespace.

Reply:
Also do you think we should update `EJBTransformers` to handle this new attributes?

Reply:
I have changed it so that we only add an argument. For now I have added it to 5.0. TBH not sure what is policy here. I will change it if necessary. I have added the transformers as well.

Reply:
The policy is any change, no matter how small, in the management API between WildFly major releases means the management API version increases by 1.0 for the next release. Same for xsd version if the xml changes. Normally the change affects both so both increase, but some changes might only affect one or the other.

This policy allows us to fork a new release stream starting from any WildFly major release and then we can use minor versions bumps of the mgmt API / xml schema if there are changes in that release stream.

Also, increasing by 1.0 is a KISS policy, eliminating debates about whether the change means a major or minor or micro bump.

Reply:
Please do any change to increase the versions in a separate commit. That provides a clean history where the actual change to the API is kept separate from the boilerplate for the version increment.

Reply:
@bstansberry  I have added the schema-update commit